website/docs/spec/registry.md

```diff
@@ -108,6 +108,28 @@ The IDL definition of the type specified by `name` along with all dependent type
 - a line containing exactly 80 `=` characters, then
 - A line containing only `IDL: <package resource name>` for that definition. The space between `IDL:` and the package resource name is mandatory. The package resource name does not include a file extension.
 
+### omgidl
+
+The `omgidl` schema encoding stores [OMG IDL](https://www.omg.org/spec/IDL/4.2/About-IDL) definitions for (X)CDR-encoded messages.
+
+- `name`: The globally-scoped name of the message type, eg. `my_company::my_module::MyType`
+- `encoding`: `omgidl`
+- `data`: valid OMG IDL definition file contents with a definition for the message type and all referenced types.
+
+#### Producing `schema.data` for the `omgidl` schema encoding
+
+Most commonly, IDL definitions for separate types are stored in separate files, and use C++ preprocessor `#include` statements to include definitions for types referenced in that definition.
```

Comment:
```suggestion
Most commonly, IDL definitions for separate types are stored in separate files, and use the preprocessor directive `#include` to include definitions for types referenced in that definition.
```

For full precision I think this language is more correct. The IDL spec never mentions "C++" in the context of the preprocessor, it just defines what a "preprocessor directive" looks like for IDL. You could call them "C++-style" if you wanted to make the connection for users, but I don't think it's necessary - though I'd also note in that case that it's really just "C-style".

Reply:
I believe the IDL spec specifically calls out C++, so i'm mirroring that here.
> IDL shall be preprocessed according to the specification of the preprocessor in ISO/IEC 14882:2003.
https://www.iso.org/standard/38110.html

Reply:
Ah - thanks for the info! I glossed over the ISO number when I read that section of the IDL spec.